verilog/analysis/verilog_linter.cc

```diff
@@ -356,12 +398,24 @@ void GetLintRuleDescriptionsHelpFlag(std::ostream* os,
   }
 }
 
-void GetLintRuleDescriptionsMarkdown(std::ostream* os) {
+void GetLintRuleDescriptionsMarkdown(std::ostream* os,
+                                     const CustomCitationMap& citations) {
   auto rule_map = analysis::GetAllRuleDescriptionsMarkdown();
   for (const auto& rule_id : analysis::kDefaultRuleSet) {
     rule_map[rule_id].default_enabled = true;
   }
 
+  if (!citations.empty()) {
```

Comment:
Same here with the loop: if 'citations' is empty, the loop will finish immediately anyway, so no extra wrapping in !empty() needed.